### PR TITLE
prep for nb2

### PR DESCRIPTION
**Pull request by zeeshanlakhani**
- [x] should do some doc work here once we're officially on nb2

### CODE REVIEW

File: docker.mk [hunk 1]
```diff
@@ -105,6 +105,8 @@ build-rustils:
 
 build-sandbox:
 	@docker build -f $(SANDBOX_DOCKERFILE) \
+		--build-arg DPDK_VERSION=${DPDK_VERSION} \
+		--build-arg TCPREPLAY_VERSION=${TCPREPLAY_VERSION} \
 		--build-arg RUSTUP_TOOLCHAIN=${RUST_VERSION} \
```
Comment (drunkirishcoder) [inline docker.mk]: if `RUSTUP_TOOLCHAIN` and `RUST_VERSION` are the same thing, do we need two variables?

File: rustils/Dockerfile [hunk 2]
```diff
@@ -21,6 +21,8 @@ RUN apt-get update \
     llvm-dev \
     wget \
   && curl -sSf https://sh.rustup.rs | sh -s -- -y --default-toolchain $RUSTUP_TOOLCHAIN \
+  # installing this for going back and forth testing (as futures-preview is beta)
+  && rustup install beta \
```
Comment (drunkirishcoder) [inline rustils/Dockerfile]: we likely won't be going back and forth, do we want to do this and make the image even bigger?

File: docker.mk (hunk 1 above)
Comment (zeeshanlakhani) [reply docker.mk]: I'll make them the same, i.e. toolchain.

File: rustils/Dockerfile (hunk 2 above)
Comment (zeeshanlakhani) [reply rustils/Dockerfile]: 👍

**Pull request merged**